server/filestore.go

```diff
@@ -99,6 +99,7 @@ type msgBlock struct {
 	liwsz   int64
 	index   uint64
 	bytes   uint64
+	rbytes  uint64
```

Comment:
I understand wanted to have short names, but to help reviewers, should we start to document the meaning of some short name variables that are not immediately obvious?

Reply:
Sure, line 1474 gives some ideas.

Reply:
Wrong line, is comment above blkSize() func. I did add some comments as requested.